SPECS/pmix/pmix.spec

```diff
@@ -103,13 +103,17 @@ find %{buildroot} -type f -name "*.la" -delete -print
 %files devel
 %{_datadir}/%{name}/*.supp
 %{_includedir}/pmix*.h
+%{_includedir}/pmix/
```

Comment:
Why was this added?

Reply:
In 4.1.2, the sources had a config option as below (along with the description)
  --with-devel-headers    normal PMIx users/applications do not need this
                          (pmix.h and friends are ALWAYS installed). Developer
                          headers are only necessary for authors doing deeper
                          integration (default: disabled).

This was disabled by default in mariner builds as well.
The rpm-build logs from pipeline also confirm:
level=debug msg="checking if want to install project-internal header files... no"

With 4.1.3, this configuration option has been removed altogether. As a result, these devel-headers are packaged during the build. Without this line, the build fails with "Unpackaged files".